.github/workflows/ci.yml

```diff
@@ -16,15 +16,16 @@ env:
 jobs:
   build_app:
     name: Build App
+    if: github.repository == 'documenso/documenso'
```

Comment:
@Mythie added the condition here as we will always run this workflow in our repo only, but if its not needed I will go ahead and remove it 😶